### PR TITLE
Add Azure portal dashboard template (from working branch)

### PR DESCRIPTION
**Pull request by mgblythe**
# PR Checklist

Check these items before submitting a PR... 

[Contribution Guide](https://github.com/Azure/azure-quickstart-templates/blob/master/1-CONTRIBUTION-GUIDE/README.md)

[Best Practice Guide](https://github.com/Azure/azure-quickstart-templates/blob/master/1-CONTRIBUTION-GUIDE/best-practices.md)

- [x] - Please check this box once you've submitted the PR if you've read through the Contribution Guide and best practices checklist.

## Changelog

*
*
*

### CODE REVIEW

File: 101-azure-portal-dashboard/azuredeploy.json [hunk 1]
```diff
@@ -0,0 +1,268 @@
...
+    },
+    "virtualMachineResourceGroup": {
+      "type": "string",
+      "metadata": {
+        "description": "Name of the resource group that contains the virtual machine"
+      }
+    },
+    "dashboardName": {
+      "type": "string",
+      "defaultValue": "[newGuid()]",
```
Comment (bmoore-msft) [inline 101-azure-portal-dashboard/azuredeploy.json:19]: make this idempotent, e.g.

guid(parameters('virtualMachineName'), parameters('virtualMachineResourceGroup'))

Comment (mgblythe) [reply 101-azure-portal-dashboard/azuredeploy.json:19]: Thanks @bmoore-msft - have you reviewed the rest and this is the only change, or are you still reviewing?

Comment (bmoore-msft) [reply 101-azure-portal-dashboard/azuredeploy.json:19]: finished... the rest looks good

Comment (mgblythe) [reply 101-azure-portal-dashboard/azuredeploy.json]: @bmoore-msft - made the change, should be good to go

**Pull request merged**